documentation/tutorials/buildlinux.rst

```diff
@@ -0,0 +1,83 @@
+============================================================
+  How to build the Linux kernel for an x86_64 architecture
+============================================================
+
+.. contents::
+   :depth: 1
+   :local:
+   :backlinks: none
+
+.. highlight:: console
+
+Get a Linux repository
+----------------------
+After successfully :ref:`installing kw<install-and-remove-kw>`, it's time to
+build the Linux kernel using it. Let's start by cloning the Linux repository
+from Torvalds into our current working directory::
+
+  git clone https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git
+
+Then, change the current directory to the Linux repository::
+
+  cd linux
+
+.. note::
+    From now on, all commands will assume you're in the kernel folder.
+
+Get .config
+-----------
+To build the kernel, a `.config` file is necessary. This file contains
+configurations about the kernel compilation process.
+To retrieve this file from your computer, run either::
```

Comment:
I think it would be nice to expand this a little bit, when I first tried to do this I got confused when my arch config didnt work in a ubuntu machine. Maybe a disclaimer that you need to use a .config file from the same distribution on which you want to use the kernel on?